11ty/definitions/dumb.md

```diff
@@ -8,18 +8,30 @@ defined: true
 excerpt: non-verbal, non-speaking, silent, refraining from speaking or unable to speak; sometimes __mute__ which can also be ableist; colloquially used to mean unintelligent or senseless
 speech: adjective
 alt_words:
-  - basic
-  - incomprehensible
-  - nonsensical
-  - redundant
-  - shallow
-  - silly
-  - senseless
-  - strange
-  - weird
-  - unfamiliar
-  - unfathomable
-  - unnecessary
+  dumb (noun):
+    - basic
+    - incomprehensible
+    - nonsensical
```

Comment:
Should we avoid recommending any language connecting senses or intelligence to lack of knowledge?

Reply:
I get what you're saying—my understanding is nonsense is connotes sense in the meaning of reason. From what I've read particularly from Deaf folks, it seems like there's a desire to not overcorrect (i.e., avoid words like "I see what you mean". With that said, I'd rather err on the side of caution so I can remove it, but wanted to provide the context!

Reply:
Yeah, I've heard similar things from Deaf folks. It feels like maybe there's an heuristic here where we avoid using it as an adjective but are OK with it as a noun? Eg. "that's nonsense" seems alright to me, but "you're being nonsensical" implies that potentially ableist comparison. What do you think?

Reply:
I can see that! Either way I agree we should remove!